### ISSUE TITLE
Format date with 2-digits doesn't work properly on IE9 and IE10

### ISSUE THREAD

**Issue by bcecchinato** [andyearnshaw/Intl.js#105]
Hi there !

I'm using ember-intl on one of my project and found a small bug under IE10 and IE9.
My hbs file is : 
```hbs
<p>{{format-date news.publicationBeginDate format="date-style"}}</p>
```

And my format.js file : 
```js
export default {
    'date-style': {
      day:  '2-digit',
      month:  '2-digit',
      year:  'numeric'
    }
};
```

I'm expecting to get a date like 01/01/2015, but when the first digit of day/month is 0, the rendered date is 1/1/2015.

Since version 1.0.0-rc-1, Chrome and IE11 are ok§. IE9 and IE10 are displaying the wrong date.

Regards,

**Comment by caridy** [comment]
this one should be easy to fix :), I hope!

**Comment by caridy** [comment]
you can now try intl@1.0.0-rc-4, should be fixed. feel free to reopen.

**Comment by bcecchinato** [comment]
Hi @caridy !

I havn't been able to test the version 1.0.0-rc-4 as it is not properly release : 
```
bower pretender#0.1.0                       validate 0.1.0 against https://github.com/trek/pretender.git#0.1.0
bower intl#~1.0.0-rc-4                  ENORESTARGET No tag found that was able to satisfy ~1.0.0-rc-4

Additional error details:
Available versions: 1.0.0-rc-1, 0.1.4, 0.1.3, 0.1.2, 0.1.1, 0.1.0, 0.0.3, 0.0.2, 0.0.1
```

Can you make a proper release build ?
